### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'warnings.length') - Version 0.58.4

### ISSUE THREAD

**Issue by kailashvele** [facebook/react-native#28588]
Have tried Stackoverflow as well, No solution.

This used to work, I have many other Apps of the same Version and they all are working fine. A couple of months back created a new app with the same react-native init command and its working fine. Now don't know what changed.

## Description
TypeError · TypeError: undefined is not an object (evaluating 'warnings.length') This error is located at: in YellowBoxListRow (at YellowBoxList.js:91) in RCTView (at View.js:45) in View (at VirtualizedList.js:1734) in CellRenderer (at VirtualizedList.js:687) in RCTView (at View.js:45) in View (at ScrollView.js:873) in RCTScrollView (at ScrollView.js:1002) in ScrollView (at VirtualizedList.js:1061) in VirtualizedList (at FlatList.js:625) in FlatList (at YellowBoxList.js:87) in RCTView (at View.js:45) in View (at YellowBoxList.js:79) in YellowBoxList (at YellowBox.js:104) in YellowBox (at AppContainer.js:93) in RCTView (at View.js:45) in View (at AppContainer.js:115) in AppContainer (at renderApplication.js:33)

## React Native version:
React Native 0.58.4

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. react-native init ProjectName --version=0.58.4
2. cd ProjectName
3. react-native run-android

## Expected Results
App should have executed and shown the screen

## Snack, code example, screenshot, or link to a repository:
Mac OS Catalina
Latest Android Studio Version
Latest Android OS on OnePlus6T

**Comment by LiangXuxuxu** [comment]
hellow,how did you resolve it?

**Comment by kailashvele** [comment]
I didn't, still waiting for some insightful comments from the maintainers. Would be helpful to understand the cause. Would be willing to help raise a PR if I understand the challenge.

**Comment by tracyhyq** [comment]
I encountered the same problem，😫

**Comment by jorgeBeck** [comment]
Hey guys, same problem here.
In the meanwhile, you can at least run and test the app by using the **Debug JS Remotely**.

**Comment by kailashvele** [comment]
But it has edge cases when you make the Release build. And you can't find the problem because it will not be the same case when Debug mode is on.

**Comment by kailashvele** [comment]
Sorry, but the whole objective is to work on the desired version, upgrading will be later part of development this year. Hence the issue is raised. This wasn't the case a month back it was fine then.

1. And Yes it works on the higher version, for now, I have used 0.59
2. Removed the 0.58.4 installation so the environment info cant is shared, but it's straight forward to recreate the challenge I have mentioned in the description.

**Comment by ibonkonesa** [comment]
Same here :(

**Comment by lilulu2018** [comment]
Met the same problem

**Comment by kailashvele** [comment]
Hi Team,

More cases are going to be reported on the same, there has to be some solution to this. Not all will be able to upgrade their react-native versions.

Hi Whoever is facing this challenge,
The closest version that works in 0.59 and changing to 0.59 has helped me resolve it without breaking changes. I hope it works for you too.

**Comment by LiangXuxuxu** [comment]
thanks! I have resolved the problem from upgrading my RN version



At 2020-04-16 12:57:58, "Kai" <notifications@github.com> wrote:

Sorry, but the whole objective is to work on the desired version, upgrading will be later part of development this year. Hence the issue is raised. This wasn't the case a month back it was fine then.

And Yes it works on the higher version, for now, I have used 0.59
Removed the 0.58.4 installation so the environment info cant is shared, but it's straight forward to recreate the challenge I have mentioned in the description.
